openapi.yaml

```diff
@@ -1673,6 +1760,50 @@ components:
         - parent_names
         - props
         - ranking_factors
+    CalendarResponse:
+      type: object
+      properties:
+        entries:
+          description: The entries of the requested
+          type: array
+          items:
+            $ref: '#/components/schemas/CalendarEntry'
+        last_sync:
+          description: When the last sync with TUMonline happened.
+          type: string
+          format: date-time
+          example: '2018-01-01T00:00:00'
+      required:
+        - entries
+        - last_sync
+    CalendarEntry:
+      type: object
+      properties:
+        title:
+          description: The title of the Entry
+          type: string
+          example: 'Quantenteleportation'
+        entry_type:
+          description: The title of the Entry
+          type: string
```

Comment:
Is this correct as string? I was just able to test the API, and my responses look like this for example:

```json
{
    "title": "Experimentalphysik 3",
    "start": "2022-02-07T08:30:00",
    "end": "2022-02-07T10:00:00",
    "entry_type": {"Lecture": "Vorlesung"}
}
```

Reply:
Okay, there are two ways to fix this:
I have correctly doccumented the current behaviour.
The problem is, that for some `entry_type`'s which may be interesting for the user while some others do not.
What do you think about this approach? instead?
![image](https://user-images.githubusercontent.com/26258709/215233301-c9f5ff82-22c6-49df-a170-f51ec161d5ab.png)

Reply:
It makes it more cleaner I think, so I think it's better this way (as two properties of type string)